gymnasium/envs/mujoco/half_cheetah_v5.py

```diff
@@ -0,0 +1,291 @@
+    | 7   | angular velocity of second rotor     | -Inf | Inf | ffoot                            | hinge | angle (rad)              |
+    | 8   | x-coordinate of the front tip        | -Inf | Inf | rootx                            | slide | velocity (m/s)           |
+    | 9   | y-coordinate of the front tip        | -Inf | Inf | rootz                            | slide | velocity (m/s)           |
+    | 10  | angle of the front tip               | -Inf | Inf | rooty                            | hinge | angular velocity (rad/s) |
+    | 11  | angle of the second rotor            | -Inf | Inf | bthigh                           | hinge | angular velocity (rad/s) |
+    | 12  | angle of the second rotor            | -Inf | Inf | bshin                            | hinge | angular velocity (rad/s) |
+    | 13  | velocity of the tip along the x-axis | -Inf | Inf | bfoot                            | hinge | angular velocity (rad/s) |
+    | 14  | velocity of the tip along the y-axis | -Inf | Inf | fthigh                           | hinge | angular velocity (rad/s) |
+    | 15  | angular velocity of front tip        | -Inf | Inf | fshin                            | hinge | angular velocity (rad/s) |
+    | 16  | angular velocity of second rotor     | -Inf | Inf | ffoot                            | hinge | angular velocity (rad/s) |
+    | excluded |  x-coordinate of the front tip  | -Inf | Inf | rootx                            | slide | position (m)             |
+
+
+    ## Rewards
+    The total reward is: ***reward*** *=* *forward_reward - ctrl_cost*.
+
+    - *forward_reward*:
+    A reward for moving forward,
+    this reward would be positive if the Half Cheetah moves forward (in the positive $x$ direction / in the right direction).
+    $w_{forward} \times \frac{dx}{dt}$, where
+    $dx$ is the displacement of the "tip" ($x_{after-action} - x_{before-action}$),
+    $dt$ is the time between actions which is depends on the `frame_skip` parameter (default is 5),
+    and `frametime` which is 0.01 - so the default is $dt = 5 \times 0.01 = 0.05$,
+    $w_{forward}$ is the `forward_reward_weight` (default is $1$).
+    - *ctrl_cost*:
+    A negative reward to penalize the Half Cheetah for taking actions that are too large.
+    $w_{control} \times \\|action\\|_2^2$,
+    where $w_{control}$ is the `ctrl_cost_weight` (default is $0.1$).
+
+    `info` contains the individual reward terms.
+
+
+    ## Starting State
+    The initial position state is $\mathcal{U}_{[-reset\_noise\_scale \times 1_{9}, reset\_noise\_scale \times 1_{9}]}$.
+    The initial velocity state is $\mathcal{N}(0_{9}, reset\_noise\_scale^2 \times I_{9})$.
+
+    where $\mathcal{N}$ is the multivariate normal distribution and $\mathcal{U}$ is the multivariate uniform continuous distribution.
+
+
+    ## Episode End
+    #### Termination
+    The Half Cheetah never terminates.
+
+    #### Truncation
+    The maximum duration of an episode is 1000 timesteps.
```

Comment:
Should we change this to `The default duration of an episode is 1000 timesteps` as the number of timesteps can be modified using `gymnasium.make(..., max_episode_steps=5)`